wordpress-mobile/WordPress-iOS
# Deselect start-over cell on settings screen

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/13307

Fixes a visual glitch where after returning from the Start Over screen (Site | Settings | Start Over),  the tapped cell remains highlighted.

### Testing steps

1. Open the Start Over modal (Site | Settings | Start Over)
2. Tap the close button to close the modal
3. Observe that the tapped cell is no longer highlighted.

### Demo

<kbd><img width="250" src="https://user-images.githubusercontent.com/1898325/81127256-adb5b380-8f0b-11ea-8c62-935ad4a54a68.gif" /></kbd>



PR submission checklist:
